public/javascripts/SVValidate/src/label/LabelVisibilityControl.js

```diff
@@ -12,6 +12,10 @@ function LabelVisibilityControl () {
     let labelVisibilityButtonOnPano = $("#label-visibility-button-on-pano");
     let labelDescriptionBox = $("#label-description-box");
 
+    let isAmsterdam = false;
+    if (labelDescriptionBox.attr('class') === "label-description-box amsterdam") {
```

Comment:
Instead of setting the correct class in the HTML and then figuring out the correct city from that, can you just pass the relevant data directly to the JavaScript code? You can do this by including the data in the `param` object in `validation.scala.html`. Then `param` gets fed into `Main.js` and then it can be passed through to `LabelVisibilityControl()`.

Reply:
This is intended to make the code more straightforward and probably less confusing by passing the data directly to where it needs to go!